### DISCUSSION TITLE
2024/02/14/acm/codeforces/CodeforcesRound913/

### DISCUSSION THREAD

**Discussion by giscus** [Hukeqing/Hukeqing.github.io#10] [General(讨论贴)]
# 2024/02/14/acm/codeforces/CodeforcesRound913/

Codeforces Round 913 (Div. 3) 个人写题记录

https://blog.mauve.icu/2024/02/14/acm/codeforces/CodeforcesRound913/

**Comment by kernelbin** [comment]
大佬带带我qaq

**Reply by Hukeqing** [reply]
呜呜呜，应该是你带带我

**Reply by kernelbin** [reply]
你带带我！

**Reply by Hukeqing** [reply]
不哦，是你带带我哦～
